client/lib/pages/main_pages/check_up_poster_page.dart

```diff
@@ -12,6 +12,7 @@ import 'package:who_app/components/page_scaffold/page_scaffold.dart';
 import 'package:who_app/components/themed_text.dart';
 import 'package:who_app/constants.dart';
 
+// TODO: Remove if unused.
```

Comment:
Remove this comment as we'll be keeping the `check_up_poster` around for certain countries where we can't get a check-up tool approved.